api/proto/disperser/disperser.proto

```diff
@@ -89,13 +96,44 @@ message DisperseBlobRequest {
 	string account_id = 3;
 }
 
+message DispersePaidBlobRequest {
+	// The data to be dispersed.
+	// The size of data must be <= 2MiB. Every 32 bytes of data chunk is interpreted as an integer in big endian format
+	// where the lower address has more significant bits. The integer must stay in the valid range to be interpreted
+	// as a field element on the bn254 curve. The valid range is 
+	// 0 <= x < 21888242871839275222246405745257275088548364400416034343698204186575808495617
+	// containing slightly less than 254 bits and more than 253 bits. If any one of the 32 bytes chunk is outside the range, 
+	// the whole request is deemed as invalid, and rejected. 
+	// NOTE: I want to include dataLength here, not the data itself.
+	bytes data = 1;
+	// The quorums to which the blob will be sent, in addition to the required quorums which are configured
+	// on the EigenDA smart contract. If required quorums are included here, an error will be returned.
+	// The disperser will ensure that the encoded blobs for each quorum are all processed
+	// within the same batch. The request doesn't need to include the payment split because the information is registered on-chain.
+	// In theory the quorum numbers should be the same as the ones in the DisperseBlobRequest, but I'm allowing freedom 
+	// for individual requests.
+	repeated uint32 custom_quorum_numbers = 2;
+	
+	// The account ID of the client. This should be a hex-encoded string of the ECSDA public key
+	// corresponding to the key used by the client to sign the BlobAuthHeader.
+	string account_id = 3;
+
+	// The bin index of the blob.
+	uint32 bin_index = 6;
+	// The cumulative payment of the blob.
+	uint64 cumulative_payment = 7;
```

Comment:
should we use [common.PaymentHeader](https://github.com/Layr-Labs/eigenda/blob/master/api/proto/common/common.proto#L32)?

Reply:
oh! didn't realize this was merged. Going to do a more general update to reflect the changes